src/main/antlr4/Expr.g4

```diff
@@ -0,0 +1,125 @@
+grammar Expr;
```

Comment:
I can't remember if antlr can add comments, but if it can, it would be great if you split everything in 3 sections

- high level expression/equation stuff
- the "base" node/atom stuff like number
-  constant stuff like `LPAREN`